docarray/array/mixins/io/pushpull.py

```diff
@@ -250,6 +250,8 @@ def _get_chunk(_batch):
         if response.ok:
             return response.json()['data']
         else:
+            if response.status_code == 403:
+                response.reason = response.json()['message']
```

Comment:
The `403` error is not the only one. How about trying to get a message from every error response?
All errors should have a message, but checking for its existence is also helpful to have.